pipeline/authz/remote_json.go

```diff
@@ -40,11 +49,23 @@ type AuthorizerRemoteJSON struct {
 
 // NewAuthorizerRemoteJSON creates a new AuthorizerRemoteJSON.
 func NewAuthorizerRemoteJSON(c configuration.Provider) *AuthorizerRemoteJSON {
-	return &AuthorizerRemoteJSON{
+	authorizer := &AuthorizerRemoteJSON{
 		c:      c,
 		client: httpx.NewResilientClientLatencyToleranceSmall(nil),
 		t:      x.NewTemplate("remote_json"),
 	}
+	// Add configurable HTTP client if set in config
+	conf, _ := authorizer.Config(nil)
```

Comment:
not sure when this would return an error - but should this error be handled here? (guessing maybe not because of checks on line 59)